docs/scos/dev/technical-enhancement-integration-guides/integrating-category-cms-blocks.md

```diff
@@ -17,295 +17,184 @@ related:
-7. Set the block to active to use it straight away.
-8. After clicking **Save**, you will be prompted to provide glossary keys for the placeholders  included in the Twig template.
-9. Embed the block into the category page by adding the following code in the `catalog.twig` template:
+1. In the Backoffice, go to the Content section and navigate to the Blocks section.
+2. Click Create Block to create a new block.
+3. From the template drop-down, select the "Title and description block" template and name the new block.
+4. Select the Categories in the "Categories: Top", "Categories: Middle" and "Categories: Bottom" fields. While typing, the product search will offer suggestions from the product list.
+5. After clicking **Save**, you will be prompted to provide glossary keys for the placeholders  included in the Twig template.
+6. Set the block to active to use it straight away.
+7. Embed the block into the category page by adding the following code in the `catalog.twig` template:
 
 ```twig
 {% raw %}{%{% endraw %} if category is defined {% raw %}%}{% endraw %}
-	{% raw %}{{{% endraw %} spyCmsBlock({category: category.id}) {% raw %}}}{% endraw %}
+	{% raw %}{{{% endraw %} spyCmsBlock({category: category.id_category}) {% raw %}}}{% endraw %}
 {% raw %}{%{% endraw %} endif {% raw %}%}{% endraw %}
 ```
 
-10. To see the page in Yves, the client data storage (Redis) must be up-to-date. This is handled through `cronjobs`.
-11. To manually execute this step,  run the collectors to update the frontend data storage:
-
-```bash
-vendor/bin/console collector:storage:export
-```
+8. To see the page in Yves, the client data storage (Redis) must be up-to-date. This is handled through `cronjobs`.
 
 **To configure block positions**
 
 Usually you don't want to change Twig templates for each block assignment, but still be able to manage CMS Blocks from Zed GUI. In this case we recommend to use positioning.
 
 CMS Block positioning means that you can predefine some of the useful places in your Twig templates once and then manage your CMS Blocks based on relations to categories and position. For example, you could define "header", "body", "footer" positions to manage your CMS Blocks in those places independently.
 
-By default we provide the following positions: "Top", "Middle", "Bottom", but you can easily change them in the module configuration on a project level (put your extension of `CmsBlockCategoryConnectorConfig` with the replaced method `getCmsBlockCategoryPositionList` to `Pyz\Zed\CmsBlockCategoryConnector\CmsBlockCategoryConnectorConfig` as in the example below).
+By default, we provide the following positions: "Top", "Middle", "Bottom", but you can easily change them in the module configuration on a project level (put your extension of `CmsBlockCategoryConnectorConfig` with the replaced method `getCmsBlockCategoryPositionList` to `Pyz\Zed\CmsBlockCategoryConnector\CmsBlockCategoryConnectorConfig` as in the example below).
```

Comment:
Please divide this into several sentences. The second sentence should be a direct step to achieve the goal